internal/hcl/evaluator.go

```diff
@@ -184,17 +177,24 @@ func (e *Evaluator) evaluateModules() {
 
 		vars := module.Definition.Values().AsValueMap()
 		moduleEvaluator := NewEvaluator(
-			e.projectRootPath,
-			module.Path,
+			Module{
+				Name:       module.Definition.FullName(),
```

Comment:
Maybe better to rename this variable to `moduleCall` as `module.Module` looks strange? Or does it look okay for you?

Reply:
yup agreed